src/fixvec/fixvec.spec.ts

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable import/extensions */
+import { serializeFixvec, deserializeFixvec } from '.'
+
+const { serialize: serializeFixture, deserialize: deserializeFixture } = require('./fixture.json')
+
+describe('Test serialize fixvec', () => {
+  const fixtureTable: [
+    any[],
+    string | undefined,
+    string | undefined,
+  ][] = serializeFixture.map(
+    ({ source, expected, exception }: { source: any[]; expected?: string; exception?: string }) => [
+      source,
+      expected,
+      exception,
+    ],
+  )
+
+  test.each(fixtureTable)(`%s => %s ? %s`, (source, expected, exception) => {
+    if (exception) {
+      expect(() => serializeFixvec(source)).toThrow(exception)
+    } else {
+      const actual = serializeFixvec(source)
+      expect(actual).toBe(expected)
+    }
+  })
+})
+
+describe('Test deserialize array', () => {
```

Comment:
Maybe test deserialize fixvec

Reply:
fixed in https://github.com/Keith-CY/molecule-javascript/pull/15/commits/92429b8897265d9d2d4dbe2fb662d457400c2ea9